src/components/MultiHopTrade/components/FiatRamps/FiatRampTradeBody.tsx

```diff
@@ -43,58 +56,98 @@ const fiatInputButtonProps = {
   px: 2,
 }
 
-const fiatQuickAmounts = ['$100', '$300', '$1,000']
+const fiatQuickAmounts = [
```

Comment:
nitpick: should say USD

Reply:
Also, are we aware that this currently doesn't generalise?

<img width="576" height="664" alt="Image" src="https://github.com/user-attachments/assets/48f5c81c-7459-408a-bcbc-92735c3574f4" />

<img width="553" height="679" alt="Image" src="https://github.com/user-attachments/assets/e914a514-85d7-4f82-9b11-344e0ae034c5" />

Reply:
I think I changed this somewhere, need to check after the end of the review

re: looks fine now after all the changes